### PR TITLE
[excel] (Preview APIs) Updating the Excel preview API table

### PR DESCRIPTION
**Pull request by AlexJerabek**
This content is made from the current type definitions for office-js and office-js-preview in DefinitelyTyped.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[6b42ce7](https://github.com/OfficeDev/office-js-docs-pr/commits/6b42ce72f198758b833fd934df39603be82151b2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/excel-preview-apis.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-NewTable/docs/reference/requirement-sets/excel-preview-apis.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202003162323075806-1672%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=g9SzpFl9isdDyEkIINIst5neEAb7qvw7o%2FDpRZFy3DE%3D&st=2020-03-16T23%3A19%3A19Z&se=2020-04-16T23%3A24%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/requirement-sets/excel-preview-apis.md [hunk 1]
```diff
@@ -30,32 +31,85 @@ The following table lists the Excel JavaScript APIs currently in preview. To see
...
+||[type](/javascript/api/excel/excel.tablefilteredeventargs#type)|Gets the type of the event. See Excel.EventType for details.|
+||[worksheetId](/javascript/api/excel/excel.tablefilteredeventargs#worksheetid)|Gets the id of the worksheet which contains the table.|
 |[Workbook](/javascript/api/excel/excel.workbook)|[close(closeBehavior?: Excel.CloseBehavior)](/javascript/api/excel/excel.workbook#close-closebehavior-)|Close current workbook.|
 ||[save(saveBehavior?: Excel.SaveBehavior)](/javascript/api/excel/excel.workbook#save-savebehavior-)|Save current workbook.|
 ||[use1904DateSystem](/javascript/api/excel/excel.workbook#use1904datesystem)|True if the workbook uses the 1904 date system.|
-|[Worksheet](/javascript/api/excel/excel.worksheet)|[customProperties](/javascript/api/excel/excel.worksheet#customproperties)|Gets a collection of worksheet-level custom properties.|
+|[Worksheet](/javascript/api/excel/excel.worksheet)|[customProperties](/javascript/api/excel/excel.worksheet#customproperties)|Returns a collection of worksheet-level custom properties.|
```
Comment (Rick-Kirkham) [inline docs/reference/requirement-sets/excel-preview-apis.md]: Why the switch from "Gets" to "Returns"?

**Comment by o365devx** [comment]
Docs Build status updates of commit _[0d8e7f0](https://github.com/OfficeDev/office-js-docs-pr/commits/0d8e7f0d91842f18bd12ffc18f63c8dd1b9091a1)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-f2d6d71c)

<a id="f2d6d71c"></a>
###
 - **[Error]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/OfficeDev/office-js-docs-pr, working directory: W:\bpmm-s, target branch: master, target branch commit: 6f0c5e39d64dbecbcb98a7148e62f01212501659, source commit: 0d8e7f0d91842f18bd12ffc18f63c8dd1b9091a1.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C19%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202003191914164871-1672%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=USru%2F%2FxPctCF9X2syaNRSfNQdDjco7tQM2cpuC4zGhQ%3D&st=2020-03-19T19%3A09%3A29Z&se=2020-04-19T19%3A14%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
Thanks @Rick-Kirkham. I've made the changes and filed a work item to fix them upstream.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[9f2585d](https://github.com/OfficeDev/office-js-docs-pr/commits/9f2585d8904dba648c0f8f9050326a42d62ed47b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/excel-preview-apis.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-NewTable/docs/reference/requirement-sets/excel-preview-apis.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C19%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202003191916224201-1672%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=aAH6uAwgcmldZxT%2BUVwQOK0N9s1iuTAu9%2Bd7x7ecg%2BM%3D&st=2020-03-19T19%3A12%3A35Z&se=2020-04-19T19%3A17%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**